# Information for a single detected object
geometry_msgs/Pose pose
string label
float32 confidence
int32 class_id